### PR TITLE
refactor: approach

### PR DESCRIPTION
**Pull request by AlefrankM**

### CODE REVIEW

File: src/contracts/ContractsAddreses.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const CONTRACTS_ADDRESSES: {
+  [key: string]: { LedaAddress: string; MarketplaceAddress: string };
+} = {
+  local: {
+    LedaAddress: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+    MarketplaceAddress: '0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0',
+  },
+  dev: {
+    LedaAddress: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
```
Comment (GimerStackIt) [inline src/contracts/ContractsAddreses.ts:9]: I think these are not the addresses we are using for dev.

File: src/contracts/ContractsAddreses.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const CONTRACTS_ADDRESSES: {
+  [key: string]: { LedaAddress: string; MarketplaceAddress: string };
+} = {
+  local: {
```
Comment (mmiranda-stackit) [inline src/contracts/ContractsAddreses.ts:4]: Maybe we should not persist this local data, that is main point of this refactor

**Review by GimerStackIt — COMMENTED** [review]
These are the addresses:
Marketplace: 0xd66411A87d80793b8F7C281479780B420e396305
LedaNFT: 0xc915858E7fCe9E6cB2690fa983ba6F8D53eC75A3

File: src/contracts/ContractsAddreses.ts (hunk 2 above)
Comment (AlefrankM) [reply src/contracts/ContractsAddreses.ts:4]: The previous PR fix this problem

**Review by juliswer — APPROVED** [review]
Looks good for me. Please check the ABIs are correct before merging

**Pull request closed without merging**